content/applications/general/voip/voip_widget.rst

```diff
@@ -0,0 +1,76 @@
+===========
+VoIP widget
+===========
+
+Phone calls
+===========
+
+Make phone calls by clicking on the :guilabel:`📞 (phone)` icon in the top navigation bar. A widget
+appears in the lower left corner of the Odoo dashboard. Freely navigate throughout the database
+while making and receiving calls from the widget.
+
+When receiving call in Odoo, the :abbr:`VoIP (Voice over Internet Protocol)` widget rings and
+displays a notification. To minimize the widget click the :guilabel:`X` icon in the upper right of
+the screen.
+
+.. note::
+   The :abbr:`VoIP (Voice over Internet Protocol)` number is the one provided by Axivox. It can be
+   accessed by navigating to `https://manage.axivox.com/ <https://manage.axivox.com/>`_. After
+   logging into the portal go to :menuselection:`Users --> Outgoing number` (column).
+
+.. image:: voip_widget/call.png
+   :align: center
+   :alt: VoIP call in Odoo.
+
+Troubleshooting
+===============
+
+.. tip::
+   If a *Missing Parameter* message appears in the **Odoo softphone**, refresh the Odoo window and
+   try again.
+
+   .. image:: voip_widget/missing-parameter.png
+      :align: center
+      :alt: "Missing Parameter" error message in the Odoo softphone.
+
+.. tip::
+   If an *Incorrect Number* message appears in the Odoo softphone, make sure to use the
+   international format, leading with the plus (+) sign followed by the international country code.
+   E.g., +16506913277 (where +1 is the international prefix for the United States).
+
+   .. image:: voip_widget/incorrect-number.png
+      :align: center
+      :alt: "Incorrect Number" error message in the Odoo softphone.
+
+Tabs
+====
+
+Recent
+------
+
+Under the :guilabel:`Recent` tab of the :abbr:`VoIP (Voice over Internet Protocol)` widget, one can
+see all the call history for the user. This includes incoming and outgoing calls. Any number can be
+clicked into to call back.
+
+Next activities
+---------------
+
+Under the :guilabel:`Next Activities` a user can see any activities assigned to them that a due to
+be completed for the day. Click in the activity to perform any actions including: Sending an email,
+accessing their contact, scheduling another activity or accessing a linked record (such as a Sales
+Order, Lead/Opportunity or Project Task). The user can also mark the activity as complete, edit it,
+or cancel it out.
+
+The customer can be called simply by pressing the :guilabel:`📞 (phone)` icon, or clicking on the
+:guilabel:`⌨️ (keyboard)` icon to dial another number for the customer.
+
+.. image:: voip_widget/activity-widget.png
+   :align: center
+   :alt: Activity control center on the VoIP widget.
+
+Contacts
+--------
+
+Under the :guilabel:`Contacts` tab, a user can access a contact in the *Contacts* app. Any contact
+can be easily called by clicking into the contact. A search feature is also available in the upper
+right-side of the widget, represented by a :guilabel:`🔍 (magnifying glass)` icon.
```

Comment:
```suggestion
Under the :guilabel:`Recent` tab of the *VoIP* widget, the call history for the user is available.
This includes incoming and outgoing calls. Any number can be clicked to begin a call.

Next activities
---------------

Under the :guilabel:`Next Activities` tab of the *VoIP* widget, a user can see any activities
assigned to them, and are due to be completed for the day.

Click an activity from this tab to perform any actions including: Sending an email, accessing their
contact, scheduling another activity, or accessing a linked record (such as a Sales Order,
Lead/Opportunity, or Project Task).

The user can also mark the activity as complete, edit the details of the activity, or cancel it.

To call the customer related to a scheduled activity, click the :guilabel:`📞 (phone)` icon, or click
the :guilabel:`⌨️ (keyboard)` icon to dial another number for the customer.

.. image:: voip_widget/activity-widget.png
   :align: center
   :alt: Activity control center on the VoIP widget.

Contacts
--------

Under the :guilabel:`Contacts` tab of the *VoIP* widget, a user can access a contact in the
*Contacts* app.

Any contact can easily be called by clicking into the contact from the *VoIP* widget's
:guilabel:`Contacts` tab.

A search feature is also available in the upper right-side of the widget, represented by a
:guilabel:`🔍 (magnifying glass)` icon.
```
please make sure all screenshots being used here, and throughout this PR, are for 15.0. Necessary adjustments and new screenshots can be added for version updates that'll happen in future PRs

Also, on the tabs of the widget, it might be nice to use emojis (when applicable) and 1-2 sentence explanations of what each icon means/represents, per the usual RST routine 👍 

Also, for this rst file (mainly)...I think, since the VoIP module is close to an app, *asterisks* should be used. But, if you think it would be better to use what's actually seen on the pop-up widget (:guilabel:`VOIP`) that seems suitable, as well. That's completely your call. Whatever you choose, however, make sure it is used throughout to keep everything looking nice and uniform. Thanks 👍

Reply:
I've combed through and updated all images.

Reply:
Added in asterisks where necessary on module mentions

Reply:
There are no emojis on the tabs, so a little confused about what you mean by that comment.

Reply:
Made changes and added in icons